src/chart/sankey/SankeyView.js

```diff
@@ -274,8 +286,19 @@ export default echarts.extendChartView({
                 el.cursor = 'move';
             }
 
+            el.highlight = function () {
+                this.trigger('emphasis');
+            };
+
+            el.downplay = function () {
+                this.trigger('normal');
+            };
```

Comment:
Should better not mount method dynamically to an instance.
It's fine just call `el.trigger`